BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.h

```diff
@@ -46,6 +47,12 @@
 @property (nonatomic, readonly, assign) BOOL appToAppBoxAuthenticationEnabled;
 
 /**
+ * This property reflects whether the client should cache API request responses. 
+ * Its default is NO.
+ */
+@property (nonatomic, readwrite, assign) BOOL requestCachingEnabled;
```

Comment:
Suggestion: ```@property (nonatomic, readwrite, assign, getter=isRequestCachingEnabled) BOOL requestCachingEnabled```. Apple has never been the best at auto-generating the proper grammar for bool checks.